apps/desktop/src-tauri/src/backend.rs

```diff
@@ -20,123 +22,192 @@ fn get_device_id() -> String {
 
 static SENTRY: OnceCell<ClientInitGuard> = OnceCell::new();
 
-pub(super) fn bleep<R>(app: &mut App<R>) -> plugin::Result<()>
+#[tauri::command]
+fn get_last_log_file(config: tauri::State<Configuration>) -> Option<String> {
```

Comment:
to be able to call it, it has to be registered in apps/desktop/src-tauri/src/main.rs line 43